.github/dependabot.yml

```diff
@@ -4,30 +4,6 @@ version: 2
 updates:
   # GitHub Action update PRs
   - package-ecosystem: "github-actions"
-    directory: "/"  # Checks /.github/workflows/
+    directory: "/"  # This is recursive so it also checks .github/workflows/
```

Comment:
I think that isn't why this work.

https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#directory

> For GitHub Actions, you do not need to set the directory to /.github/workflows. Configuring the key to / automatically instructs Dependabot to search the /.github/workflows directory, as well as the action.yml / action.yaml file from the root directory.

Reply:
Ah, interesting!

I guess that would mean those two definitions would both try to update the docs requirements which isn't what we want.

Reply:
Maybe what the docs says is a subset of "directories will be searched recursively".